web/src/settings_components.ts

```diff
@@ -1768,3 +1768,33 @@ export function set_custom_time_inputs_visibility(
         $time_select_elem.parent().find(".custom-time-input-container").hide();
     }
 }
+
+export function get_group_assigned_realm_permissions(group: UserGroup): {
+    subsection_heading: string;
+    assigned_permissions: AssignedGroupPermission[];
+}[] {
+    const group_assigned_realm_permissions = [];
+    for (const {subsection_heading, settings} of settings_config.realm_group_permission_settings) {
+        const assigned_permission_objects = [];
+        for (const setting_name of settings) {
+            const setting_value = realm[realm_schema.keyof().parse("realm_" + setting_name)];
+            const assigned_permission_object =
+                group_permission_settings.get_assigned_permission_object(
+                    group_setting_value_schema.parse(setting_value),
+                    setting_name,
+                    group.id,
+                    current_user.is_admin,
```

Comment:
Aren't there a few permissions settings that only owners can edit? I think we can fix that in a follow-up: it may require extending `settings_config.realm_group_permission_settings` with a flag indicating owners-only-can-edit.

Reply:
Confirmed. The failure mode seems not worth blocking merging this PR for, but please do a follow-up PR for this fix. 

![image](https://github.com/user-attachments/assets/8d8fd2da-e2d7-4074-a7b1-45f5694126bb)